OrbitQt/servicedeploymanager.cpp

```diff
@@ -94,6 +93,9 @@ ServiceDeployManager::ServiceDeployManager(const DeploymentConfiguration* deploy
 
   background_thread_.start();
   moveToThread(&background_thread_);
+  QMetaObject::invokeMethod(
```

Comment:
maybe duplicate the explanation here as well? "Event loop needs to be initialized in background thread to work properly on Linux"?

Reply:
Yep, good idea. On it.

Reply:
done.